include/ginkgo/core/matrix/dense.hpp

```diff
@@ -817,6 +819,35 @@ class Dense
      */
     void compute_norm1(LinOp* result, array<char>& tmp) const;
 
+    /**
+     * Create a submatrix from the original matrix.
+     * Warning: defining stride for this create_submatrix method might cause
+     * wrong memory access. Better use the create_submatrix(rows, columns)
+     * method instead.
+     *
+     * @param rows     row span
+     * @param columns  column span
+     * @param stride   stride of the new submatrix.
+     */
+    std::unique_ptr<const Dense> create_submatrix(const span& rows,
```

Comment:
```suggestion
    std::unique_ptr<const Dense> create_const_submatrix(const span& rows,
```
maybe use create_const_submatrix to make explict call like others `get_values/get_const_value`